src/design-tokens/mixins.css

```diff
@@ -48,11 +48,6 @@
     ) !important;
     background-color: var(--eds-theme-color-background-brand-primary-strong);
   }
-
-  &:visited {
-    color: var(--eds-theme-color-text-neutral-subtle);
-    text-decoration-color: var(--eds-theme-color-text-neutral-subtle);
-  }
```

Comment:
This is the crux of the PR. All the other changes are related to the randomized urls I had in the stories.